react/features/watchrtc/middleware.ts

```diff
@@ -0,0 +1,55 @@
+import { AnyAction } from 'redux';
+
+import { IStore } from '../app/types';
+import { LIB_WILL_INIT } from '../base/lib-jitsi-meet/actionTypes';
+
+import { isRtcstatsEnabled } from '../rtcstats/functions';
+import { isWatchRTCEnabled } from './functions';
+
+import logger from './logger';
+
+import MiddlewareRegistry from '../base/redux/MiddlewareRegistry';
+
+import WatchRTCHandler from './WatchRTCHandler';
+
+/**
+ * Middleware which intercepts lib-jitsi-meet initialization and conference join in order init the
+ * watchRTC.
+ *
+ * @param {Store} store - The redux store.
+ * @returns {Function}
+ */
+MiddlewareRegistry.register((store: IStore) => (next: Function) => (action: AnyAction) => {
+    const { getState } = store;
+    const state = getState();
+    const config = state['features/base/config'];
+    const { watchRTCConfigParams } = config;
+
+    switch (action.type) {
+    case LIB_WILL_INIT: {
+        if (isRtcstatsEnabled(state)) {
+            logger.error("Cannot initialize WatchRTC when RTCStats is enabled.");
```

Comment:
This will be printed for all users using rtcstats, isn't it?
If this is the case, please drop it.